src/stream-kokkos/ADD-Kokkos.cpp

```diff
@@ -0,0 +1,65 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2017-20, Lawrence Livermore National Security, LLC
+// and RAJA Performance Suite project contributors.
+// See the RAJAPerf/COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#include "ADD.hpp"
+#if defined(RUN_KOKKOS)
+#include "common/KokkosViewUtils.hpp"
+#include <iostream>
+
+namespace rajaperf {
+namespace stream {
+
+void ADD::runKokkosVariant(VariantID vid,
+                           size_t RAJAPERF_UNUSED_ARG(tune_idx)) {
+  const Index_type run_reps = getRunReps();
+  const Index_type ibegin = 0;
+  const Index_type iend = getActualProblemSize();
+
+  ADD_DATA_SETUP;
+
+  // Instiating views using getViewFromPointer
+
+  auto a_view = getViewFromPointer(a, iend);
+  auto b_view = getViewFromPointer(b, iend);
+  auto c_view = getViewFromPointer(c, iend);
+
+  auto add_lam = [=](Index_type i) { ADD_BODY; };
```

Comment:
extra lambda

Reply:
I chucked this, and the rest of the extra lambdas.